impeller/compiler/BUILD.gn

```diff
@@ -47,6 +47,10 @@ impeller_component("compiler_lib") {
     # All third_party deps must be reflected below in the impellerc_license
     # target.
     "//third_party/inja",
+
+    # This library is also used by the Flutter engine and should be included
+    # in the license set.
+    "//third_party/rapidjson",
```

Comment:
Oops. My bad. I forgot that there's already a JSON serializer in impellerc coming in with inja. It's already used here: https://github.com/flutter/engine/blob/main/impeller/compiler/reflector.cc#L192

Reply:
Done